spring-rabbit-stream/src/test/java/org/springframework/rabbit/stream/listener/RabbitListenerTests.java

```diff
@@ -99,6 +100,7 @@ void nativeMsg(@Autowired RabbitTemplate template) throws InterruptedException {
 	}
 
 	@Test
+	@Disabled("Temporary until SF uses Micrometer snaps")
```

Comment:
I don't think we need this since this PR is going to be merged already after respective change in the SF.

Reply:
We need it (temporarily) so that the PR build will succeed.